src/Console/ConsoleStyle.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Console;
+
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class ConsoleStyle extends SymfonyStyle
+{
+    /**
+     * @param string[] $elements
+     */
+    public function listing(array $elements): void
+    {
+        $elements = array_map(function ($element) {
+            return sprintf(' - %s', $element);
+        }, $elements);
+        $this->writeln($elements);
+        $this->newLine();
+    }
+}
```

Comment:
Nice move! Personally I'd use compsition over inheritance, but in this case it would create over 100 lines of duplicated code.

So :+1:  for this one